src/providers/AppCheckerProvider/docs/AppCheckerProvider.stories.mdx

```diff
@@ -0,0 +1,42 @@
+<Meta title="SDK Providers/AppCheckerProvider" />
+
+# AppCheckerProvider
+
+The `AppCheckerProvider` provides functionality to check meeting readiness.
+
+## Importing
+
+```jsx
+import { AppCheckerProvider } from 'amazon-chime-sdk-component-library-react';
+```
+
+## Usage
+
+`AppCheckerProvider` needs to be used with by `MeetingProvider` and `DevicesProvider`.
+
+```jsx
+import React from 'react';
+import {
+  MeetingProvider,
+  DevicesProvider
+  AppCheckerProvider,
+  useAppCheckerManager
+} from 'amazon-chime-sdk-component-library-react';
+
+const App = () => (
+  <MeetingProvider>
+    <DevicesProvider>
+      <AppCheckerProvider>
+        <MyChild />
+      </AppCheckerProvider>
+    </DevicesProvider>
+  </MeetingProvider>
+);
+
+const MyChild = () => {
+  // When a meeting session is started, you'll have access to the appCheckerManager instance
```

Comment:
I'm a little confused - I remember Anuran saying that the AppChecker needs to be tied to its own dummy meeting session, and not the meeting session that we are actually trying to join. What happens right now when a user joins a meeting, goes to the checker to check the comparability, then goes back to resume the meeting?